evm/src/witness/traces.rs

```diff
@@ -0,0 +1,32 @@
+use crate::arithmetic::columns::NUM_ARITH_COLUMNS;
+use crate::cpu::columns::CpuColumnsView;
+use crate::logic;
+use crate::witness::memory::MemoryOp;
+
+type LogicRow<T> = [T; logic::columns::NUM_COLUMNS];
+type ArithmeticRow<T> = [T; NUM_ARITH_COLUMNS];
+
+struct Traces<T: Copy> {
```

Comment:
nit: could do `<F: Field>` since we'll only compute traces over the base field; `Copy` might seem like a hint that we'll use it in other ways